colossus/src/main/scala/colossus/controller/OutputController.scala

```diff
@@ -285,4 +291,13 @@ trait OutputController[Input, Output] extends MasterController[Input, Output] {
     }
   }
 
+  def idleCheck(period: Duration): Unit = {
```

Comment:
I can certainly sync the access (I don't think there's a better way of making it threadsafe) but my understanding was it wasn't needed here.